packages/amplify-e2e-core/src/categories/api.ts

```diff
@@ -23,6 +23,9 @@ import { selectRuntime, selectTemplate } from './lambda-function';
 import { modifiedApi } from './resources/modified-api-index';
 
 export function getSchemaPath(schemaName: string): string {
+  if (process.env.CODEBUILD_SRC_DIR) {
+    return path.join(process.env.CODEBUILD_SRC_DIR, 'packages', 'amplify-e2e-tests', 'schemas', schemaName);
+  }
   return path.join(__dirname, '..', '..', '..', 'amplify-e2e-tests', 'schemas', schemaName);
 }
```

Comment:
any idea why `__dirname` didn't work? `__dirname` should point to absolute path of the current file (i.e. api.js after compilation)

Reply:
I could not figure out why it did not work. I did verify from the .cast files that the tests are running using the amplify.exe binary.

Reply:
did it work on linux ? if this is just for windows please add comments in both places explaining that.
and perhaps add `ifWindows` condition to the mix.